src/test/modelmanager_test.cpp

```diff
@@ -484,6 +484,26 @@ TEST(ModelManager, ConfigReloadingWithWrongInputName) {
     ASSERT_EQ(status, ovms::StatusCode::CONFIG_SHAPE_IS_NOT_IN_NETWORK);
 }
 
+TEST(ModelManager, ConfigReloadingStatefulDynamic) {
```

Comment:
Test here strongly relies on the verification order of model config. Define correct model config (probably even load such model) and then just replace both the stateful & batching mode to have incorrect config.

The same for the test below.